### ISSUE TITLE
User request: Android support

### ISSUE THREAD

**Issue by elimisteve** [cryptag/notes#49]

**Comment by elimisteve** [comment]
There's definitely significant interesting in this...

**Comment by jaduncan** [comment]
Yes. A web interface is good, but a major advantage to Keep is that I can look at notes offline. That said, it would also be possible to make this a progressive web app with offline caching just in the browser. That's in some ways better, because you get Android and iOS all in one at the same time that you can add offline caching in FF and Chrome on the desktop.

**Comment by jaduncan** [comment]
(and, with a responsive design, all in the same codebase)
